website/blog/2021-02-04-64updates.md

```diff
@@ -0,0 +1,98 @@
+
+# 🍎 React Native for macOS
+Community Work
+
+- React-Native-Community macOS ProgressView support
+- React-Native-Community macOS Clipboard support
+- React-Native-Community macOS SVG Icon support
+- We can’t check this in because the repo owner has to merge it and he is just one person who is “taking a break” from OSS, but it’s up and running in our fork.
+We’re setting up the npm publishing CI this week for macOS
+
+- macOS keyboarding events support
+- macOS Tooltip support for Button, Text, Image, Slider, and TextInput
+This is out for review now but will get checked in tomorrow
+
+# 🚀 TurboModules
+
+We've added support for C++ JSI TurboModules as they exist today in React Native core projects. You won't have to worry about writing any code differently from how you do today in React Native, but when you run your app on Windows, we will ensure it works over the ABI-safe boundaries. This means your React Native Windows TurboModules can be shipped in different DLLs and we take care of the binary compatibility between versions.
+
+This feature is still considered experimental in 0.64, because it is not finished in React Native Core. Meaning we may need to change the ABI surface in future releases.
+
+# ✨ Developer Goodness
+As with many of our releases, there's always a number of developer and "under the covers" upgrades we make to the framework as a whole. These usually includes things like performance improvements, better API parity and of course general bug fixes, but let's take a look at some of the notable items available this release.
+
+## 🎯 Easy opt-in to Hermes
+
+Performance is a huge deal when developing in general, but especially so when it comes to using React Native. We're pretty invested is using Hermes and while there are teams using it in production, support is still experimental and we do not have the full Hermes debugger or Hermes in C# capabilities online yet. We would still love to know if you try Hermes and run into any issues, so go ahead and [give it a try today!]((https://microsoft.github.io/react-native-windows/docs/next/hermes))
+
+## 🎉 More API parity
+
+More component properties supported by the other platforms now work on Windows! These include Text's `backgroundColor`, `border`, and `textTransform`, as well as `Platform.version` and an initial version of `AccessibilityInfo` are now available.
+
+To get the full list of release details, including breaking changes, check out our [release notes over on our repo.](https://github.com/microsoft/react-native-windows/releases/tag/react-native-windows_v0.64.0-preview.1)

```

Comment:
I think it would be nice to do a deeper dive into framework additions, breaking changes, etc.

E.g. upstream 0.63 blog post is here. It's very focused on the framework itself, and what those updating need to know. https://reactnative.dev/blog/2020/07/06/version-0.63

Reply:
I agree - might be a good idea to make it slightly more technical and perhaps reshuffle the sections to reflect what we consider the most important additions over the last few months

Reply:
I'd imagine that @kikisaints wouldn't mind some help producing content on these topics 😉

Reply:
@NickGerleman added/rearranged per your feedback. When you get a chance, take a look and I'll refine from there.

Reply:
This is looking realy good 🙂

Hermes has been a hot-button issue for some developers. Consider putting that one above some of the other examples.

Reply:
I missed it skimming through, but I do think we need to find a place to call out breaking changes. It's probably the first thing I've gotten asked about for partners upgrading.

Think it would be fine to copy-paste from release noes.

Reply:
Sounds good! I'll add that and beef up the Hermes part.